nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/resources/org/apache/nifi/web/webdefault.xml

```diff
@@ -551,6 +551,24 @@
             <http-method-omission>TRACE</http-method-omission>
         </web-resource-collection>
     </security-constraint>
-
+    <filter>
```

Comment:
I agree with @mcgilman 's question about why we can't define and associate this filter in the `JettyServer` class for consistency.

Reply:
it was an earlier difficulty, now overcome.